react/modules/customPixelEvents.ts

```diff
@@ -0,0 +1,40 @@
+import { CustomPixelEventInterface } from '../interfaces/custom-pixel-event.interface'
+
+/**
+ * Fires custom pixel event
+ *
+ * @param {string} eventName - Event name
+ * @param {mixed} [eventData] - Event data
+ */
+export const triggerPixelEvent = (
+  eventName: string,
+  eventData?: CustomPixelEventInterface
+): void => {
+  if (typeof window === 'undefined' || !('postMessage' in window)) return
+
+  const origin = '*'
+
+  const data = {
+    ...eventData,
+    eventName,
+  }
+
+  window.postMessage(data, origin)
+}
+
+/**
+ * Fires custom pixel event for newsletter
+ *
+ * @param {CustomPixelEventInterface} [eventData] - Event additional data
+ */
+export const triggerPixelNewsletterEvent = (
```

Comment:
These booth functions will be used in other apps?!

Reply:
@igorbrasileiro No, this is the old implementation, i`ve removed the files